sats-dna/src/main/kotlin/com/sats/dna/components/card/SatsArticleCard.kt

```diff
@@ -25,67 +29,101 @@ import com.sats.dna.components.button.SatsDismissButton
 import com.sats.dna.theme.SatsTheme
 import com.sats.dna.tooling.PreviewImagePlaceholder
 
+/**
+ * @param imageUrl URL of the background image.
+ * @param title The title of the article.
+ * @param subtitle The subtitle of the article.
+ * @param onClick Click callback function.
+ * @param modifier Modifier to apply to the card.
+ * @param isFixed If true, gives a fixed background and text color.
+ * @param tag Optional tag to display on top of the card.
+ * @param dismissButton Optional dismiss button to display on top of the card.
+ */
 @Composable
 fun SatsArticleCard(
     imageUrl: String,
-    title: String,
+    title: String?,
     subtitle: String?,
     onClick: () -> Unit,
     modifier: Modifier = Modifier,
+    isFixed: Boolean = false,
     tag: @Composable (() -> Unit?)? = null,
     dismissButton: @Composable (() -> Unit?)? = null,
 ) {
     SatsCard(modifier) {
-        Column(Modifier.clickable { onClick() }) {
-            Box {
-                HeroImage(imageUrl)
-                Row(
-                    modifier = Modifier.fillMaxWidth(),
-                    verticalAlignment = Alignment.CenterVertically,
-                ) {
-                    tag?.let {
-                        Box(Modifier.padding(SatsTheme.spacing.s)) {
-                            tag()
-                        }
+        Box(
+            Modifier
+                .height(234.dp)
+                .clickable { onClick() },
+        ) {
+            HeroImage(imageUrl, modifier = Modifier.fillMaxSize())
+            Row(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .align(Alignment.TopStart),
+                verticalAlignment = Alignment.CenterVertically,
+            ) {
+                tag?.let {
+                    Box(Modifier.padding(SatsTheme.spacing.s)) {
+                        tag()
                     }
+                }
 
-                    Spacer(Modifier.weight(1f))
+                Spacer(Modifier.weight(1f))
 
-                    dismissButton?.let {
+                dismissButton?.let {
+                    Box(Modifier.padding(SatsTheme.spacing.xxs)) {
                         dismissButton()
                     }
                 }
             }
 
+            val (backgroundColor, textColor) = if (isFixed) {
+                SatsTheme.colors.surfaces.fixed.primary.default.bg to SatsTheme.colors.surfaces.fixed.primary.default.fg
+            } else {
+                SatsTheme.colors.surfaces.primary.default.bg to SatsTheme.colors.surfaces.primary.default.fg
+            }
```

Comment:
We could potentially get a `ColorSet` back here instead:


```kotlin
val colors = if (isFixed) {
    SatsTheme.colors.surfaces.fixed.primary.default
} else {
    SatsTheme.colors.surfaces.primary.default
}
```

That allows using `colors.bg` and `colors.fg`. Not saying it's better, just saying it's possible. 😅

Reply:
Fixed in e7333fe2712369ed4800ee088d139ece1a0074bc